src/App.js

```diff
@@ -1,14 +1,38 @@
-import logo from './logo.svg';
+import React, { useState } from 'react';
+
 import './App.css';
+//import nav bar and container components for react boostrap
+import Nav from 'react-bootstrap/Nav';
+import Navbar from 'react-bootstrap/Navbar';
+import Container from 'react-bootstrap/Container';
```

Comment:
I believe this can be written as 
`import { Nav, Navbar , Container } from 'react-bootstrap';`

Reply:
Ahh yes, good to know, will be much cleaner as we add more bootstrap components!